packages/nextjs/src/utils.ts

```diff
@@ -12,3 +13,21 @@ export const refresh = async (nhostUrl: string, refreshToken: string): Promise<N
   if (result.ok) return result.json()
   else return Promise.reject(result.statusText)
 }
+
+export const NHOST_SESSION_KEY = 'nhostSession'
+
+export const setNhostSessionInCookie = (param: NhostClient | NhostSession | null) => {
+  const session = param && 'auth' in param ? param.auth.getSession() : param
+  if (session) {
```

Comment:
Maybe it's just nitpicking but changing this to

```ts
if (!session) {
  Cookies.remove(NHOST_SESSION_KEY)
  
  return
}

const { refreshToken, ...rest } = ...
```

makes the code so much nicer.